app/javascript/packs/provider.ts

```diff
@@ -20,7 +19,7 @@ document.addEventListener('DOMContentLoaded', () => {
     .forEach(form => {
       form.addEventListener('change', () => {
         if (form.dataset.remote) {
-          void jQuery1.rails.handleRemote(jQuery1(form))
+          void window.Rails.handleRemote(jQuery1(form))
```

Comment:
this `Rails` thing is something UJS adds?

Reply:
Yes, `jquery_ujs` embedded it inside `window.$`, whereas `rails-ujs` declare in its own global scope.